### PR TITLE
fix(filter): make tree age filter work

### PR DESCRIPTION
**Pull request by dnsos**
The PR addresses the bug that no trees were displayed when changing the age filter on non-mobile devices (mobile devices were not affected because they pull a different dataset).

The bug was caused by an inexistent key `standalter` that we were trying to assign to our `age` field in the GeoJSON property. The key that **is** existent in the data source is called `age` as well.

The [fix](https://github.com/technologiestiftung/giessdenkiez-de/blob/114740744af19b55f92dcd44f3d732021b9cf5c1/src/utils/requests/loadTreesGeoJson.ts#L33) in this PR is to change the key from `standalter` to `age`. Also, some typing is added, so that confusion is hopefully mitigated in the future.

### CODE REVIEW

File: src/common/interfaces.ts [hunk 1]
```diff
@@ -70,7 +70,8 @@ export interface Tree {
   hausnr?: string | null;
   zusatz?: string | null;
   pflanzjahr?: string | null;
-  standalter?: string | null;
+  standalter?: string | null; // this may be redundant now, but I'm not entirely sure
```
Comment (vogelino) [inline src/common/interfaces.ts:73]: I searched for this property everywhere in the codebase and I think we can remove it altogether. Good catch! 🪲🔫

File: src/components/TreesMap/DeckGLMap.tsx [hunk 2]
```diff
@@ -156,19 +158,19 @@ class DeckGLMap extends React.Component<DeckGLPropType, DeckGLStateType> {
...
-            radolan_sum: number;
-            age: number;
-          };
+          properties: TreeGeojsonFeatureProperties;
         }): [number, number, number, number] => {
           const { ageRange, mapViewFilter, communityData } = this.props;
           const { properties } = info;
           const { id, radolan_sum, age } = properties;
-          const communityDataFlagMap = communityData && communityData[id];
+          const communityDataFlagMap = communityData && id && communityData[id];
```
Comment (vogelino) [inline src/components/TreesMap/DeckGLMap.tsx:166]: This is still called flagMap... But that's not a problem really ⛳️

File: src/components/TreesMap/DeckGLMap.tsx [hunk 3]
```diff
@@ -156,19 +158,19 @@ class DeckGLMap extends React.Component<DeckGLPropType, DeckGLStateType> {
...
           const { isWatered, isAdopted } = communityDataFlagMap || {};
           const transparent = [0, 0, 0, 0] as [number, number, number, number];
 
+          if (!radolan_sum || !age) {
+            return transparent;
```
Comment (vogelino) [inline src/components/TreesMap/DeckGLMap.tsx:171]: 👍  Very good! To bugs killed with one stone 🥌🐛🐞

**Review by vogelino — APPROVED** [review]
Great bug hunting! Well done!

File: src/common/interfaces.ts (hunk 1 above)
Comment (dnsos) [reply src/common/interfaces.ts:73]: Hmm, I've taken another look now and the type is apparently needed in `src/components/Sidebar/SidebarTree/TreeInfos.tsx`. Removing it strangely doesn't throw errors in dev mode, but I would be hesitant to remove it already...

**Pull request merged**